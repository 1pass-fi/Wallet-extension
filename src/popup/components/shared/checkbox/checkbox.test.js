import { render, fireEvent } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import React from 'react'

import Checkbox from '.'

describe('Test for Checkbox component', () => {
  describe('Non-empty label', () => {
    it('render correctly', () => {
      const {container} = render(<Checkbox label='label' />)
      expect(container).toMatchSnapshot('non-empty_label')
    })
  }) 
})
