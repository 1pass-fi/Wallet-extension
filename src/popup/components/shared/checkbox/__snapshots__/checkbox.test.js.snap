// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Checkbox component Non-empty label render correctly: non-empty_label 1`] = `
<div>
  <div
    class="checkbox-container undefined"
  >
    <div
      class="checkbox-input"
    >
      <input
        class="checkbox-hidden"
        type="checkbox"
      />
      <div
        class="checkbox-styled"
        style="background: rgb(255, 255, 255);"
      >
        <div
          class="check-mark-icon"
          style="visibility: hidden;"
        >
          <svg />
        </div>
      </div>
    </div>
    <span
      class="checkbox-label"
    >
      label
    </span>
  </div>
</div>
`;
